roles/etcd/defaults/main.yaml

```diff
@@ -4,8 +4,10 @@ r_etcd_common_backup_sufix_name: ''
 
 l_is_etcd_system_container: "{{ (openshift_use_etcd_system_container | default(openshift_use_system_containers | default(false)) | bool) }}"
 
+l_etcd_static_pod: "{{ not (r_etcd_common_skip_command_shim is defined and r_etcd_common_skip_command_shim) or openshift.node.bootstrapped }}"
```

Comment:
please tell me bootstrapped didn't make it's way into openshift_facts ?

Reply:
Its being set in playbooks/init/cluster_facts.yml:
```
    openshift_facts:
      role: node
      local_facts:
        sdn_mtu: "{{ openshift_node_sdn_mtu | default(None) }}"
        bootstrapped: "{{ openshift_is_bootstrapped }}"

```